templates/telemetry_derived/experiments_v1/get_experiment_list.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+"""Get experiment list for a given date from the recipe server."""
+
+from argparse import ArgumentParser
+import json
+import sys
+import time
+
+import requests
+
+experiments_qualifying_actions = {"preference-experiment", "opt-out-study"}
```

Comment:
I have a more extensive revision to this script I'd like to submit later so we can use the v3 api and do backfills correctly, but for now this will fix bug 1587231, if you don't mind rolling this in
```suggestion
experiments_qualifying_actions = {"preference-experiment", "opt-out-study", "branched-addon-study", "multi-preference-experiment"}
```